amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -546,9 +547,16 @@ type MotionMessage @model {
   initiatorAddress: ID!
   name: String!
   messageKey: String!
+  motionId: ID!
```

Comment:
Is that the database or native motion id?

Reply:
It's the database motion id